PlayerMarkerModule.cs

```diff
@@ -31,10 +32,16 @@ public class PlayerMarkerModule : Module
 		public SettingEntry<float> SettingPlayerMarkerOpacity { get; private set; }
 		public SettingEntry<float> SettingPlayerMarkerRadius { get; private set; }
 		public SettingEntry<float> SettingPlayerMarkerVerticalOffset { get; private set; }
+		public SettingEntry<string> SettingPlayerMarkerTexture { get; private set; }
 
 		private Tortle.PlayerMarker.Controls.PlayerMarker _playerMarker;
 
-		private Texture2D _textureCircleFill;
+		public static string[] _textureDisplay = new string[] { "GW2 Target", "Solid", "Thin Ring", "Thick Ring" };
+
+		private Texture2D _texturethin;
+		private Texture2D _texturethick;
```

Comment:
Don't think we should keep these tbh. I don't see the rings being a useful preset